spec/datadog/tracing/contrib/shoryuken/tracer_spec.rb

```diff
@@ -183,6 +183,9 @@ def perform(sqs_msg, body)
       # end
 
       it do
+        # Test does not work correctly on Ruby 3.5.0-preview1.
+        # TODO: Ruby 3.5 - Re-enable after investigation.
+        skip 'Skipped on Ruby 3.5.0-preview1' if RUBY_VERSION.start_with?('3.5')
```

Comment:
```suggestion
        skip 'Skipped on Ruby 3.5.0-preview1' if RUBY_VERSION =~ /\A3\.5.*preview/
```
What do you think about this pattern to get these tests running on 3.5 final automatically?

Reply:
Uuuh I like like this -- rather than "we need to remember", actually break CI on a non-preview ruby! This is a great idea +1!

Reply:
Cool TIL about `pending`! But turns out this test was failing because it needed the `ostruct` gem (thanks @ivoanjo  https://github.com/DataDog/ruby-guild/issues/258), so I'll un-skip this entirely.